src/app.ts

```diff
@@ -38,6 +41,20 @@ class App {
   private mountRoutes(): void {
     this.app.use(router)
   }
+
+  private async createDBConnection() {
```

Comment:
create a new directory in src/db.
Create mongo.ts, redis.ts and index.ts there. Import this function from there